packages/alert-dialog/index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "@reach/alert-dialog" {
+  import { DialogProps, DialogContentProps } from "@reach/dialog";
+
+  type AlertDialogProps = {
+    isOpen?: boolean;
+    onDismiss?: () => void;
+    leastDestructiveRef?: React.RefObject<HTMLElement>;
+    children: React.ReactNode;
+  } & DialogProps;
+
+  type AlertDialogContentProps = {
+    children: React.ReactNode;
+  } & DialogContentProps;
+
+  export const AlertDialog: React.SFC<AlertDialogProps>;
```

Comment:
```suggestion
  export const AlertDialog: React.FC<AlertDialogProps>;
```
`React.SFC` is deprecated.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/98ed28785ea16dfa4a7746d2c5399716baae8cbf/types/react/index.d.ts#L457